### PR TITLE
Add a blog for the Podman 0.12.1.1 release

### PR DESCRIPTION
**Pull request by mheon**
As the title says

### CODE REVIEW

File: _posts/2018-12-12-podman-alpha-v0.12.1.1.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+This release comes with many exciting new features. To highlight a few of our biggest changes:
+
+    * The `podman generate kube` command was added by Brent Baude, which generates Kubernetes pod and service YAML from Podman containers and pods.
+	* Initial support for named volumes using the `podman volume` set of commands was landed by Urvashi Mohnani
+	* The `podman rm` and `podman rmi` commands can now prune unused containers and images with the `--prune` flag
+	* Ports can now be published to the host from pods
+
+Numerous bugs were fixed as well, including a breaking change in rootless Podman found in 0.11.x releases.
+
+To see the full changelog, please visit our release notes on [github](https://github.com/containers/libpod/blob/master/RELEASE_NOTES.md)
```
Comment (TomSweeneyRedHat) [inline _posts/2018-12-12-podman-alpha-v0.12.1.1.md]: nit, could you change "github" to "GitHub".  Ditto line 30.

File: _posts/2018-12-12-podman-alpha-v0.12.1.1.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+We're happy to announce the availability of Podman 0.12.1.1, our latest version. We've been very busy over the last month, and it shows! We've merged over 150 new commits since our 0.11 releases, including major new functionality and several critical bugfixes. Pods, Kubernetes compatability, and container volumes all saw major improvements.
+
+We hope everyone enjoys the release, and stays with us in the future as Podman gets closer to 1.0. As always, many thanks to everyone who contributed to this release!
+
+<!--readmore-->
+## Changes
+This release comes with many exciting new features. To highlight a few of our biggest changes:
+
+    * The `podman generate kube` command was added by Brent Baude, which generates Kubernetes pod and service YAML from Podman containers and pods.
```
Comment (TomSweeneyRedHat) [inline _posts/2018-12-12-podman-alpha-v0.12.1.1.md]: If you haven't yet, hit "View File", these points are in a scrollable code box and I'm thinking that's not what you meant.  Can you try having only two spaces in front of each asterisk here and see if that fixes it?

File: _posts/2018-12-12-podman-alpha-v0.12.1.1.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+	* Initial support for named volumes using the `podman volume` set of commands was landed by Urvashi Mohnani
+	* The `podman rm` and `podman rmi` commands can now prune unused containers and images with the `--prune` flag
+	* Ports can now be published to the host from pods
+
+Numerous bugs were fixed as well, including a breaking change in rootless Podman found in 0.11.x releases.
+
+To see the full changelog, please visit our release notes on [github](https://github.com/containers/libpod/blob/master/RELEASE_NOTES.md)
+
+Some of this work, like the `podman volume` command, is still very early. We'd greatly appreciate feedback! If you have an enhancement request or a bug report, please file them on our [github](https://github.com/containers/libpod).
```
Comment (TomSweeneyRedHat) [inline _posts/2018-12-12-podman-alpha-v0.12.1.1.md]: suggest:  [GitHub Issues(https://github.com/containers/libpod/issues) page.

**Comment by TomSweeneyRedHat** [comment]
Looks great over all, thanks for putting it together.  Just a couple of format issues, then it will be gtg.

**Comment by mheon** [comment]
@TomSweeneyRedHat Should be fixed - I think the formatting looks right now, but tell me if that's not what you were looking for

**Comment by TomSweeneyRedHat** [comment]
@mheon, ugh, still the same.  Let me go play on an md online editor and see if I can figure it out.  Might be the colon and/or the blank line before, trey weird.

**Comment by TomSweeneyRedHat** [comment]
@mheon it's fine, ugh! browser cache bit me in the keister. 
FWIW, I find the last two words in the the blog "our GitHub." to be a bit odd.  Can I convince to add "issue page" at the end or drop the "our"?  If not, I'll publish as is.

**Comment by mheon** [comment]
Issue page SGTM, will fix

**Comment by mheon** [comment]
Pushed a fix

**Review by TomSweeneyRedHat — COMMENTED** [review]
LGTM, thanks @mheon, I'll publish first thing tomorrow (Wed 12/12)

**Pull request merged**